ROADMAP.md

```diff
@@ -1,61 +1,55 @@
+| |Community roadmap prioritization | n/a | ongoing |
+| Enhance cloud-native user experience for skilled and new users | Unmanaged-cluster model introduced to enable minimal-cluster deployments in under 4 minutes that can run on consumer hardware. | Implementation in Progress | v0.10 |
+| | Work to reduce time to cluster availability on vSphere and public clouds (Cluster API v1.1 Proposal) | Awaiting Proposal | TBD |
+| | Consider more efficient and resource-saving methods to bootstrap | Awaiting proposal | TBD |
+| | Make management clusters registerable in Tanzu Mission Control | Implementation in progress | v0.11 |
+| | Introduce a UI that takes a user from installation through creating clusters to installing packages on clusters. Embed documentation and guidance throughout so that users can be successful exclusively through this tooling. | Awaiting Proposal | TBD |
+| | Expose Cluster API (capi)-provider bootstrapping details, client-side, to users when creating management and workload clusters. Provides users an understanding of where bootstrapping failures occurred. | Awaiting Proposal [(issue)](https://github.com/vmware-tanzu/community-edition/issues/2730 ) | TBD |
+|Robust Kubernetes platform | Limited-internet (including airgapped) support | Awaiting Proposal | TBD |
+| | Validate and improve end-to-end tests across all supported infrastructure providers. | Awaiting proposal | TBD |
+| Rich package library | Work with community to solicit and prioritize desired packages| n/a |ongoing |
+| | Pursue community and ISV package lifecycle and configuration integrations via Carvel and Tanzu Framework | n/a | ongoing |
+| Positioning Community Edition as leading-edge upstream for Tanzu products | Community Edition produces its own Bill of Materials (BOM), Tanzu Kubernetes Release (TKr), OCI container images, host images, and packages for CLI plugins and in-cluster software | Awaiting Proposals | TBD |
+| Application Platform | Installable package is made available containing pre-configured, minimally-viable, platform services that enable developers to run their tools and apps (contour, kpack, cert-manager, knative serving, Cartographer, etc.) | Implementation in Progress (TODO: tracker proposal for all components) | Beginning in v0.10; please see issue (in dev) |
+| | Tools and processes for package contribution, maintenance, and installation | Awaiting Proposal | TBD |
+| | Package maintainer documentation and guidance is live such that everyone can understand the responsibilities of being a package maintainer | Awaiting Proposal | TBD |
+| |Provide guidance on bringing new provider packages to TCE; enabling the inclusion of new infrastructure providers | Awaiting Proposal (dependent on Tanzu Kubernetes Provider implementation) | TBD |
+| |Introduce kubeapps to TCE for discovery and installation of packages from Bitnami library [(issue)](https://github.com/vmware-tanzu/community-edition/issues/2418) | Awaiting Proposal | TBD |
```

Comment:
Thought I had made this comment earlier, but this bitnami goal seems to be in opposition to this goal:

> Pursue community and ISV package lifecycle and configuration integrations via Carvel and Tanzu Framework

Won't introducing another packaging paradigm create fracture the packaging community and make it _more_ difficult to get started easily with packages and a full running platform? Yes, we have many existing assets in the Bitnami library, but in my eyes, it's risky introducing them as an _alternative_ to our established packaging philosophy.

Further, does this create the expectation that _maintainers_ of packages maintain _both_ their carvel package _and_ the bitmani chart? If I'm the packaging dev for prometheus, this seems like extra work when I've potentially already created automation, tooling, and workflows for the carvel packaging set.

I'd love @seemiller's perspective on this too.

Reply:
There are many packages in Bitnami that are not in our repos. And they are duplicating packages in Carvel tooling.  At this point there is very little overlap, and we should advise users to search the TCE repo before Kubeapps.